vllm/model_executor/layers/linear.py

```diff
@@ -97,6 +98,21 @@ def apply(self,
         Expects create_weights to have been called before on the layer."""
         raise NotImplementedError
 
+    def create_weights_moe(self, layer: torch.nn.Module, num_experts: int,
+                           hidden_size: int, intermediate_size: int,
+                           params_dtype: torch.dtype, **extra_weight_attrs):
+        raise NotImplementedError(
+            "Moe models are not yet supported for your LinearMethod")
```

Comment:
Can we use `self.__class__.name` here to replace "your LinearMethod"?

Reply:
Yes